taletime/lib/player/widgets/player_controls.dart

```diff
@@ -1,11 +1,28 @@
 import "package:audio_service/audio_service.dart";
 import "package:flutter/material.dart";
+import "package:share_plus/share_plus.dart";
+import "package:taletime/internationalization/localizations_ext.dart";
 import "package:taletime/main.dart";
 import "package:taletime/player/models/custom_player_state.dart";
 
 class PlayerControls extends StatelessWidget {
   const PlayerControls({super.key});
 
+  void _shareStory(BuildContext context) {
+    final mediaItem = audioHandler.mediaItem.value;
+
+    if (mediaItem == null) {
+      return;
+    }
+
+    final title = mediaItem.title;
+    final author = mediaItem.artist;
+    final id = mediaItem.id;
+
+    final sharedText = "$title ${AppLocalizations.of(context)!.by} $author - https://taletime-2022.web.app/shared?storyId=$id";
```

Comment:
Maybe the base url should be put in an env file or something so once we scale we don't have to swap all links by https://taletime.app/shared?storyId=$id":)